source/auth/auth.md

```diff
@@ -1660,18 +1660,21 @@ def auth(connection):
   access_token, is_cache = get_access_token()
 
   # If there is a cached access token, try to authenticate with it. If
-  # authentication fails, it's possible the cached access token is expired. In
+  # authentication fails with an Authentication error (18), 
+  # it's possible the cached access token is expired. In
   # that case, invalidate the access token, fetch a new access token, and try
```

Comment:
```suggestion
  # authentication fails with an Authentication error (18), 
  # invalidate the access token, fetch a new access token, and try
```
To clarify. I believe we always invalidate (whether expired, invalidated, or any other possibility), we do not check that it has expired and only then invalidate.